### ISSUE TITLE
Allow "-1" and "--long" together

### ISSUE THREAD

**Issue by steven-lang** [ogham/exa#586]
My use case is to have an alias `alias exa="exa -1"` since I don't want all the clutter everytime I do `exa`. 
From time to time though it is nice to know more about the file properties. Running `exa -l` then produces without any other output:
```
Option --oneline (-1) is useless given option --long (-l)
```

**Comment by mschilli87** [comment]
possible workaround:

```sh
alias exa="exa -1"
alias exal="exa -l"
```

This even saves you two keystrokes compared to `exa -l` everytime you want the `--long` option.

The major dowside I can see is if you list something with

```sh
exa [some options]
```

and the decide you want to add the `--long` option, you can't simply run

```sh
!! -l
```

anymore but have to use the more clumsy

```sh
exal !!:1-$
```

instead.

**Comment by NightMachinary** [comment]
```
exa() {
    local arg long=''
    for arg in "$@"
    do
        [[ "$arg" == "-l" || "$arg" == "--long" ]] && long='y'
    done
    if test -z "$long"
    then
        command exa -1 "$@"
    else
        command exa "$@"
    fi
}
```
Dup of https://github.com/ogham/exa/issues/570

**Comment by ogham** [comment]
This has been fixed and will go out in the next release (#476).
